src/components/frame/v5/pages/MembersPage/partials/SubNavigation/SubNavigation.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import Tooltip from '~shared/Extensions/Tooltip';
+import Icon from '~shared/Icon';
+import { useMembersSubNavigation } from './hooks';
+import styles from './SubNavigation.module.css';
+
+const SubNavigation = () => {
```

Comment:
Please, add the displayName value and use FC to create a React component as in other files.

Reply:
Added